etomica-apps/src/main/java/etomica/AlkaneEH/SpeciesAlkaneEH.java

```diff
@@ -9,83 +9,86 @@
 import etomica.atom.AtomLeafDynamic;
 import etomica.atom.AtomType;
 import etomica.atom.IAtom;
-import etomica.chem.elements.ElementSimple;
 import etomica.molecule.IMolecule;
 import etomica.molecule.Molecule;
 import etomica.space.Space;
 import etomica.species.Species;
+
 /**
  * Species for TraPPE-Explicit hydrogen,Siepmann
- * 
- * @author shu
- * 01-30-2013
+ *
+ * @author shu 01-30-2013
  */
-  public class SpeciesAlkaneEH extends Species  {
+public class SpeciesAlkaneEH extends Species {
 
-	  	private static final long serialVersionUID = 1L;
-	    protected final Space space;
-	    protected final int numCarbons;
-    protected final int totalAtoms;
+    public final int numCarbons;
+    public final int numH;
+    public final int numCH2;
+    protected final Space space;
+    private final int totalAtoms;
+    private final AtomType c_3Type, c_2Type, hType;
     public int indexC_3_1;//CH3 on the left
```

Comment:
it looks like indexC_3_1 and _2 do not need to be fields at all

Reply:
Do you mean they can be methods or that they don't have to exist at all? [ConformationAlkaneEH](https://github.com/etomica/etomica/blob/a4ec260974cf842bf3d1198377f1c0d4d573fda6/etomica-apps/src/main/java/etomica/virial/ConformationAlkaneEH.java#L43) uses them, although that class isn't used anywhere.

Reply:
Ah, indeed.  I'm pretty sure Shu used ConformationAlkaneEH to collect data; she probably just didn't commit the main class.  And I don't know why it wants the species to tell it where 2 of the carbons are but has everything else hardcoded.... oh well.

Reply:
Ah right it already gets numCarbons from the species in the constructor. Do you think we can get the main class? If nothing else just so we don't delete this stuff on accident.

Reply:
er, ConformationAlkaneEH is used on 52 of this class...

Reply:
Huh there's actually a ConformationAlkaneEH in both etomica.AlkaneEH and etomica.virial. I don't know if they're functionally the same or not.

Reply:
etomica.virial.ConformationAlkaneEH is very different.  In Shu's files, there is only etomica.virial.ConformationAlkaneEH_backup and etomica.virial.ConformationAlkaneEH_backkkup.  She probably committed the stuff in virial, then copied it to etomica.AlkaneEH (and committed those), but didn't cvs delete the stuff from etomica.virial.  So, let's delete the unused EH stuff etomica.virial.